config/overlays/openshift/manager-patch.yaml

```diff
@@ -10,3 +10,4 @@
     - '--openshift-create-dashboard=true'
     - '--enable-cr-metrics=true'
     - '--create-sm-operator-metrics=true'
+    - '--feature-gates=operator.sidecarcontainers.native'
```

Comment:
Shouldn't this be prefixed with `+` ? 
e.g.  `--feature-gates=+operator.sidecarcontainers.native`